core/src/main/scala/workflows/Canopus.scala

```diff
@@ -37,7 +37,7 @@ object Canopus extends Workflow[Unit] {
       i <- DockerOp.extract(Versioned.unwrap(vunit)).inject[WorkflowOp]
       _ <- status(id, Pending, "Canopus workflow about to start")
       _ <- logToFile(id, s"Instructing ${dc.name}'s scheduler to handle service container")
-      l <- launch(i, dc, ns.name, vunit, p, hash)
+      l <- launch(i, dc, ns.name, vunit, p, None, hash)
```

Comment:
So somehow here we need to connect the dots to get the blueprint passed in, right?

Reply:
I imagine this will be implemented as part of https://github.com/getnelson/nelson/issues/86